sdks/go/pkg/beam/runners/vet/testpipeline/staticcheck.conf

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+checks = ["inherit", "-U1000"]
```

Comment:
Is there a reason to have this file located in this testpipeline directory? I suspect it is a config value that applies to the whole sdk and thus probably belongs higher in the hierarchy (maybe at the `sdks/go` level?

Reply:
Staticcheck configs can be applied per-directory. We can exempt the testpipeline code from the U1000 check (function not being used) and inherit check exemptions from higher up in the hierarchy if they exist. Eventually a top-level config would be the default checks minus the deprecation checker, but we don't want to ignore unused code in the entire Go SDK.

See https://staticcheck.io/docs/configuration/ for more info

Reply:
Ah, I see! I misunderstood the purpose - thanks!

Reply:
I'm also trying to limit how many file or line-specific lint:ignore comments we have in the codebase